mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -865,4 +865,10 @@ The new line here must be kept as the second half of the string is clickable for
 
     <!-- CFR for Toolbar Shield Icon (Banner Info Message) -->
     <string name="cfr_for_toolbar_shield_icon">Looks like this site was trying to track you. Click the shield for site settings and tracker blocking stats.</string>
+
+    <!-- Tool Tip for: Privacy and Security Settings  -->
```

Comment:
@ionutbedregeanu Please add comments in each of the strings below in order to give context to localizers (see [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#add-localization-notes) for more info).
Also, is the second string correct? It sounds like it's more of a brainstorming line :)

Reply:
@iorgamgabriel ^^